src/components/views/settings/Notifications.js

```diff
@@ -151,11 +162,16 @@ var VectorPushRulesDefinitions = {
                     "value": "default"
                 }
             ]
+        },
+        vectorStateToHsDefaultRuleEnabled: {
+            on: undefined,
+            loud: true,
+            off: false
         }
     },
 
     // When people join or leave a room
-    /*"im.vector.rule.member_event": {
+    "im.vector.rule.member_event": {
```

Comment:
Oh sorry. I believed you did not commit yet because my tool did not complained about a git conflict.
I fix it.